web/src/recent_topics_ui.js

```diff
@@ -389,6 +390,9 @@ function format_conversation(conversation_data) {
         // display / hide them according to filters instead of
         // doing complete re-render.
         context.topic_muted = Boolean(user_topics.is_topic_muted(context.stream_id, context.topic));
+        context.topic_unmuted_in_muted_stream = Boolean(
+            user_topics.is_topic_unmuted(context.stream_id, context.topic),
+        );
```

Comment:
This variable name is misleading; we haven't checked whether the stream is muted.

Reply:
It was `topic_unmuted` before but changed the name after feedback from @amanagr [here](https://github.com/zulip/zulip/pull/25140#discussion_r1171224654)(although it was a question by him).
should I change the name back to `topic_unmuted`?

Reply:
I changed it back to `topic_unmuted`; I think it's possible we will want to come up with some term for what to call this state in the codebase that we can migrate this to, but that's probably a better discussion for #frontend.